terraform/dynamodb/videos-info.tf

```diff
@@ -0,0 +1,44 @@
+provider "aws" {
+  region = "us-west-1"  # Replace with your desired region
+}
+
```

Comment:
we already have added this block in our previous code. This will give error while deploying as duplicate provider.